java/build/bazel/tests/integration/WorkspaceDriver.java

```diff
@@ -66,7 +66,7 @@ private static void setupRepositoryCache() throws IOException {
         repositoryCache.put(Paths.get(dep));
       }
     }
-    repositoryCache.freeze();
+    // repositoryCache.freeze();
```

Comment:
Rm?

Reply:
I'm waiting @ittaiz 's comment on this. If `repositoryCache.freeze()` is only for testing `testRepositoryCacheIsFrozen`, I think we can just remove it. Because I changed the url in `testRepositoryCacheIsFrozen` to a unknown host, which would produce the expected error on all platform.